service.subtitles.subtitle/resources/lib/SUBUtilities.py

```diff
@@ -377,7 +380,7 @@ def _calc_rating(self, subsfile, file_original_path):
         return round(rating, 1)
 
     def download(self, id, zip_filename):
-        # # Cleanup temp dir, we recomend you download/unzip your subs in temp folder and
+        # # Cleanup temp dir, we recommend you download/unzip your subs in temp folder and
         # # pass that to XBMC to copy and activate
         if xbmcvfs.exists(__temp__):
```

Comment:
In future updates please remove \__labelname\__ style labels. These should only be used for python internals.